core/web/resolver/mutation.go

```diff
@@ -1091,7 +1091,7 @@ func (r *Resolver) CreateJob(ctx context.Context, args struct {
 	case job.FluxMonitor:
 		jb, err = fluxmonitorv2.ValidatedFluxMonitorSpec(config, args.Input.TOML)
 	case job.Keeper:
-		jb, err = keeper.ValidatedKeeperSpec(args.Input.TOML)
+		jb, err = keeper.ValidateSpecAndCreateKeepersJob(args.Input.TOML)
```

Comment:
the "AndCreate" part of this function name feels confusing in this context since all the other job types just say "Validate"

Reply:
That's true, I will move it back to how it was then!